crates/sats/src/db/def.rs

```diff
@@ -485,6 +485,7 @@ pub struct TableSchema {
     pub sequences: Vec<SequenceSchema>,
     pub table_type: StTableType,
     pub table_access: StAccess,
+    pub scheduled: Option<Box<str>>,
```

Comment:
In the PR #1448 I'm rewriting Def. My plan was to add:

```rust
struct ScheduleDef {
    reducer_name: Box<str>,
    at_column: ColId
}
```

And then having an `Option<ScheduleDef>` on `TableDef`.
If you don't mind restructuring this that way it would make it easier for me down the line.

Reply:
How do you translate this in `TableSchema`?

Reply:
I guess just assert that `at_column` is named `at` and convert the reducer name.